<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiuOpsSubaruInput1Window</class>
 <widget class="QWidget" name="BiuOpsSubaruInput1Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Lighting Delay Time</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="light_delay_combo"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="autolock_combo"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Outside Temp Offset (degC)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Autolock Time (sec)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="outtemp_combo"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="send_setting">
      <property name="text">
       <string>Send to BIU</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
